lib/teleterm/clusters/cluster_apps.go

```diff
@@ -79,10 +79,7 @@ func (c *Cluster) getApp(ctx context.Context, authClient authclient.ClientI, app
 }
 
 // ReissueAppCert issue new certificates for the app and saves them to disk.
-func (c *Cluster) ReissueAppCert(ctx context.Context, clusterClient *client.ClusterClient, app types.Application) (tls.Certificate, error) {
-	if app.IsAWSConsole() || app.IsGCP() || app.IsAzureCloud() {
-		return tls.Certificate{}, trace.BadParameter("cloud applications are not supported")
-	}
+func (c *Cluster) ReissueAppCert(ctx context.Context, clusterClient *client.ClusterClient, routeToApp proto.RouteToApp) (tls.Certificate, error) {
 	// Refresh the certs to account for clusterClient.SiteName pointing at a leaf cluster.
 	err := clusterClient.ReissueUserCerts(ctx, client.CertCacheKeep, client.ReissueParams{
```

Comment:
This is something I'd love to remove sooner or later.

We originally added this to DB access just before the first release of Teleport Connect and then we copied it to every other kind of resource. I didn't know much about Teleport back then, Roman helped me identify this as the fix and it stuck ever since. But I don't think it should be necessary to reissue user certs each time we want to create a cert for a local proxy.